### ISSUE TITLE
Alt text is shown with "prev" or "next" when it should not show the alignment markers

### ISSUE THREAD

**Issue by jemberton** [jemberton/obsidian-booknav-plugin#1]
When any link is given, alt text is not exclusively shown. For example,

```
[[My Page | A New Name | prev]]
```

Will result in a link with the text "A New Name | prev" as the displayed text.

**Comment by jemberton** [comment]
fixed. Syntax is unchanged and allows for alternate display text using the example in original post above

**Comment by jemberton** [comment]
If regular markdown syntax for links is used, the alt text is not parsed properly.

**Comment by jemberton** [comment]
Regular markdown links should *not* be parsed for explicit alt text. Any `|` character should be considered part of the desired text for display as the nature of traditional markdown links is to use the text inside of the `[]` as the display text.
